File: loianeTraining/javaBasico/exercicios-aulas-16-17/Ex21.java
import java.util.Scanner;

public class Ex21 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite um numero");
        var number = scan.nextInt();

        var score = 0;
        for (int i = 1; i <= number; i++) {
            if (number % i == 0) {
                score++;
            }
        }
        if (score == 2) {
            System.out.println("O numero digitado e primo");
        } else {
            System.out.println("Nao e primo");
        }
        scan.close();
    }
}

File: Loiane_Training_C#/Basico/D/Ex21.cs
using System;

class Ex21
{
    static void Main(string[] args)
    {
        Console.Write("Digite um numero: ");
        int numero = int.Parse(Console.ReadLine());
        int contador = 0;
        for (int i = 1; i <= numero; i++)
        {
            if (numero % i == 0)
            {
                contador++;
            }
        }
        if (contador == 2 || numero == 1)
        {
            Console.WriteLine("Numero primo!");
        }
        else
        {
            Console.WriteLine("Numero nao primo!");
        }
    }
}